src/components/MessageList/MessageList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import TextMessage from './TextMessage/TextMessage';
+import MessageInfo from './MessageInfo/MessageInfo';
+import useVideoContext from '../../hooks/useVideoContext/useVideoContext';
+
+interface Message {
+  author: string;
+  dateCreated: Date;
+  body: string;
+  sid: string;
+  type: string;
+}
+
+interface MessageListProps {
+  messages: Message[];
+}
+
+export default function MessageList({ messages }: MessageListProps) {
+  const { room } = useVideoContext();
+  const localParticipant = room!.localParticipant;
+  return (
+    <>
+      {messages.map((message, idx) => {
+        const time = message.dateCreated
```

Comment:
Nit - I think here we can have 

`const isLocalParticipant = localParticipant.identity === message.author;`

Then `isLocalParticipant` can be passed as a prop to MessageInfo and TextMessage (and maybe MediaMessage in the future). 

This way, the `localParticipant.identity === message.author` check is done only once per message, instead of twice (once by MessageInfo and once for TextMessage).

Reply:
nice, i'll have to remember this one👍 . this also allowed me to remove the `author` prop from the `TextMessage` component since we were only using it to check whether the message author was the localParticipant